resources/lib/mvutils.py

```diff
@@ -178,6 +180,15 @@ def cleanup_filename(val):
     search = ''.join([c for c in val if c in cset])
     return search.strip()
 
+def url_lastmodified(url):
+    """
+    Query a network object's last-modified value as a unix timestamp
+
+    Args:
+        url(str): the source url of the object to query
+    """
+    with closing(urlopen(url)) as src:
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Audit url open for permitted schemes. Allowing use of file:/ or custom schemes is often unexpected.](https://app.codacy.com/manual/MediathekView/plugin.video.mediathekview/pullRequest?prid=5460169)